src/content/en/tools/workbox/guides/generate-service-worker/workbox-build.md

```diff
@@ -0,0 +1,133 @@
+project_path: /web/tools/workbox/_project.yaml
+book_path: /web/tools/workbox/_book.yaml
+description: A guide on how to generate a complete service worker with workbox-build.
+
+{# wf_blink_components: N/A #}
+{# wf_updated_on: 2018-01-25 #}
+{# wf_published_on: 2017-11-15 #}
+
+# Generate a Service Worker with workbox-build {: .page-title }
+
+{% include "web/tools/workbox/_shared/alpha.html" %}
+
+This page explains how to use the workbox-build Node module to generate a
+complete service worker with precaching and runtime caching.
+
+<aside class="note"><b>Note:</b> You'll need to have
+<a href="https://nodejs.org/en/download/">Node installed</a> to use
+workbox-build.</aside>
+
+{% include "web/tools/workbox/guides/_shared/install-workbox-build.md" %}
+
+## Call <code>generateSW()</code>
+
+To generate a service worker, you need to add `workbox-build.generateSW()`
+to your build process or Node script:
+
+```javascript
+const workboxBuild = require('workbox-build');
+
+// NOTE: This should be run *AFTER* all your assets are built
+const buildSW = () => {
+  // This will return a Promise
+  return workboxBuild.generateSW({
+    globDirectory: 'build',
+    globPatthers: [
+      '**\/*.{html,json,js,css}',
+    ],
+    swDest: 'build/sw.js',
+  });
+}
+
+buildSW();
```

Comment:
Could you remove this `buildSW()` invocation? It kind of runs counter to your earlier comment about needing to invoke the function *after* your assets are built, and you're also not making use of the promise returned by `buildSW()`.